<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Game ongoing</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarPlayer1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarPlayer2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSwitchPlayer1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>switch player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSwitchPlayer2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>switch player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSet120">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>120 sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSet300">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>300 sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>START GAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP GAME</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
